src/notation/qml/MuseScore/NotationScene/internal/PercussionPanelToolBar.qml

```diff
@@ -132,14 +176,21 @@ Item {
         orientation: Qt.Horizontal
         accentButton: true
 
-        navigation.panel: navPanel
+        navigation.panel: centralNavPanel
         navigation.row: 0
 
         onClicked: {
             root.model.finishEditing()
         }
     }
 
+    NavigationPanel {
+        id: rightSideNavPanel
+        name: "PercussionPanelToolBarRightSide"
+        section: root.navigationSection
+        order: root.navigationOrderStart + 2
```

Comment:
The navigation ought to be sequential, so it should be `order: centralNavPanel.order + 1`